task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -131,22 +172,31 @@ def test_run_deferred_basic(test_dags_dir: Path, time_machine):
     with mock.patch(
         "airflow.sdk.execution_time.task_runner.SUPERVISOR_COMMS", create=True
     ) as mock_supervisor_comms:
-        ti = parse(what)
+        ti = mocked_parse_fixture(what, "basic_deferred_run", task)
         run(ti, log=mock.MagicMock())
 
         # send_request will only be called when the TaskDeferred exception is raised
         mock_supervisor_comms.send_request.assert_called_once_with(msg=expected_defer_task, log=mock.ANY)
 
 
-def test_run_basic_skipped(test_dags_dir: Path, time_machine):
+def test_run_basic_skipped(time_machine, mocked_parse_fixture):
     """Test running a basic task that marks itself skipped."""
+    from airflow.providers.standard.operators.python import PythonOperator
+
+    task = PythonOperator(
+        task_id="skip",
+        python_callable=lambda: (_ for _ in ()).throw(
+            AirflowSkipException("This task is being skipped intentionally."),
+        ),
+    )
+
     what = StartupDetails(
         ti=TaskInstance(id=uuid7(), task_id="skip", dag_id="basic_skipped", run_id="c", try_number=1),
-        file=str(test_dags_dir / "basic_skipped.py"),
```

Comment:
Ditto.

Reply:
`basic_skipped` dag is removed in this PR itself.